hollow/src/main/java/com/netflix/hollow/api/producer/ListenerSupport.java

```diff
@@ -70,128 +74,260 @@ void remove(IncrementalCycleListener listener) {
     }
 
     void fireProducerInit(long elapsedMillis) {
-        for(final HollowProducerListener l : listeners) l.onProducerInit(elapsedMillis, MILLISECONDS);
+        for(final HollowProducerListener l : listeners) {
+            try {
+                l.onProducerInit(elapsedMillis, MILLISECONDS);
+            } catch(Throwable t) {
+                log.warning("Error executing listener" + t);
```

Comment:
Ah sorry I was unclear :( what I meant was to continue catching RuntimeException, but at the _log_ line update to something like:
`log.log(Level.WAR, "Error executing listener", e)`